config/common_versions.yml

```diff
@@ -0,0 +1,5 @@
+title: v3.17.1
+components:
+  key-cert-provisioner:
+    image: tigera/key-cert-provisioner
+    version: v1.0.0
```

Comment:
Does this image exist already? I would have though this should be set to release-v1.0

Reply:
I was just following the conventions of the other versions, I'll change it to release-v1.0.